test/MeshKernelNETTest/Api/MeshKernelTest.cs

```diff
@@ -3454,6 +3454,52 @@ public void Mesh2dExpungeStateThroughApi()
                 }
             }
         }
+
+        [Test]
+        public void Mesh2dGetNodeEdgeDataThroughApi()
+        {
+
+            // Setup
+            using (DisposableMesh2D mesh = CreateMesh2D(10, 10, 10, 10))
```

Comment:
use different numbers for input argument values to be more likely to hit errors like swapped arguments etc.